lib/oidcstrategy.js

```diff
@@ -752,4 +573,468 @@ Strategy.prototype._shouldLoadUserProfile = function shouldLoadUserProfile(issue
   return done(null, !skip);
 };
 
+/**
+ * validate id_token, and pass the validated claims and the payload to callback
+ * if code (resp. access_token) is provided, we will validate the c_hash (resp at_hash) as well
+ *
+ * @param {String} id_token
+ * @param {Object} req
+ * @param {Function} callback
+ * @param {String} code (if you want to validate c_hash)
+ * @param {String} access_token (if you want o validate at_hash)
+ */
+Strategy.prototype._validateIdToken = function validateIdToken(id_token, code, access_token, req, callback) {
```

Comment:
This method does more than id token validation. It validates code and access_token also in case of hybrid flows. Maybe validateResponse better name?